### PR TITLE
docs: add docs of "schema" in "sheet management" of chapter "FEATURES"

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2bgyr5k SDK 文档编写 —— Features > Sheet Management > Schema](https://app.clickup.com/t/2bgyr5k)

File: docs/source/features/sheet_management/schema.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
+..
+ Copyright 2022 Graviti. Licensed under MIT License.
+
+########
+ schema
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:5]: All titles should be capitalized.

File: docs/source/features/sheet_management/schema.rst [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+########
+
+Each sheet has a record type schema to describe the name and the type of each column.
+
+Graviti use `Portex`_ schema language to define the schema.
+Please refer to the `Portex documentation`_ for the schema language grammar.
+
+.. _Portex: https://github.com/Project-OpenBytes/portex
+.. _Portex documentation: https://openbytes-schema.readthedocs.io/en/latest/?badge=latest
+
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:15]: this url is useless?

File: docs/source/features/sheet_management/schema.rst [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Graviti SDK supports the python interaction with the Portex schema.
+
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:17]: should we add a concrete example of how to use these primitive types, complex types?

File: docs/source/features/sheet_management/schema.rst [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+       },
+   }
+   Vector = pt.template("Vector", vector_template)
+   vector = Vector(coords=pt.float32(), labels=["cat", "dog"])
+
+
+**************
+ schema files
+**************
+
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:138]: do we need to introduce the methods of PortexType schema?

**Review by graczhual — CHANGES_REQUESTED** [review]

File: docs/source/features/sheet_management/schema.rst [hunk 5]
```diff
@@ -4,3 +4,217 @@
...
+   enum = pt.enum(values=["a", "b", "c"])
+   enum.values
+
+.. note::
+   The values of the enum type must be strings with the format ``[A-Za-z_][A-Za-z0-9_]*``.
+
+array
+=====
+
+The array type can be created by giving the item type and length.
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:57]: here, length is not required. Should we move it into another sentence. like:
The array type can be created by giving the item type. param length can be specified to fix the length of array.

File: docs/source/features/sheet_management/schema.rst [hunk 6]
```diff
@@ -4,3 +4,217 @@
...
+
+
+record
+======
+
+The record type can be created by giving the fields, including names and types.
+The record type is used to describe the name and type of each column of the tabular data,
+and all the primitive and complex types mentioned above can be used here for each column type.
+
+The names and types can be accessed by `fields`, which acts like a dict whose key is the column name and the value is column type.
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:88]: could use `` or ref to fields

File: docs/source/features/sheet_management/schema.rst [hunk 7]
```diff
@@ -4,3 +4,217 @@
...
+
+.. code:: python
+
+   import graviti.portex as pt
+   record = pt.record([("x", pt.int32()), ("y", pt.int32()), ("categories", pt.enum(values=["cat", "dog"]))])
+   record.fields
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:94]: Format it, make it easier to read. And do we need to add a example for using dict to initialize pt.record here?

File: docs/source/features/sheet_management/schema.rst [hunk 8]
```diff
@@ -4,3 +4,217 @@
...
+
+****************
+ Schema Package
+****************
+
+Graviti SDK supports to use exteranl packages defined under a repo. By giving the repo and revision, the package can be initialized and used locally.
+
+SDK provides :func:`~graviti.portex.builder.build_package` to build an exteranl Portex type package from the repo.
+
+Take `OpenBytes`_ as an example, which is used as the standard exteranl package by Graviti.
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:166]: standard or OpenBytes?

File: docs/source/features/sheet_management/schema.rst [hunk 9]
```diff
@@ -4,3 +4,234 @@
...
+
+   # convert PyArrow type to Portex type
+   pa_type = pa.struct([pa.field("x", pa.float32()), pa.field("y", pa.float32())])
+   portex_type = pt.PortexType.from_pyarrow(pa_type)
+
+
+Expand
+======
+
+For better comprehension and operations, SDK provides methods to expand exteranl Portex type to builtin types.
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:227]: exteranl or external?

File: docs/source/features/sheet_management/schema.rst [hunk 10]
```diff
@@ -4,3 +4,234 @@
...
+
+.. code:: python
+
+   import graviti.portex as pt
+   ob = pt.build_package("https://github.com/Project-OpenBytes/standard", "main")
+   box2d = ob.label.Box2D(categories=["cat", "dog"])
+   # Expand the first layer of the exteranl type
+   internal_type = box2d.internal_type
+   # Expand the top level of the exteranl type to internal type
+   builtin_type = box2d.to_builtin()
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:237]: do we need to print `internal_type` and `builtin_type` here for better understand?

File: docs/source/features/sheet_management/schema.rst [hunk 11]
```diff
@@ -4,3 +4,234 @@
...
+.. code:: python
+
+   import graviti.portex as pt
+   ob = pt.build_package("https://github.com/Project-OpenBytes/standard", "main")
+   box2d = ob.label.Box2D(categories=["cat", "dog"])
+
+.. note::
+   Using branch as the revision to build the exteranl package is unstable, since the latest commit may change.
+
+   Tag or commit ID as revision is recommended.
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:196]: tag commit or tag name commit ID

File: docs/source/features/sheet_management/schema.rst [hunk 12]
```diff
@@ -4,3 +4,234 @@
...
+
+****************
+ Schema Package
+****************
+
+Graviti SDK supports to use exteranl packages defined under a repo. By giving the repo and revision, the package can be initialized and used locally.
+
+SDK provides :func:`~graviti.portex.builder.build_package` to build an exteranl Portex type package from the repo.
+
+Take `standard`_ as an example, which is used as the standard exteranl package by Graviti.
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:183]: exteranl or external

File: docs/source/features/sheet_management/schema.rst [hunk 13]
```diff
@@ -4,3 +4,234 @@
...
+   )
+   # Init record with dict
+   record = pt.record(
+      {
+         "x": pt.int32(),
+         "y": pt.int32(),
+         "categories": pt.enum(values=["cat", "dog"]),
+      }
+   )
+   record.fields
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:111]: do we need to print the `record` and `record.fields` here for better understand? And the array, tensor before

File: docs/source/features/sheet_management/schema.rst [hunk 14]
```diff
@@ -4,3 +4,234 @@
...
+.. _Portex: https://github.com/Project-OpenBytes/portex
+
+Graviti SDK supports the python interaction with the Portex schema.
+
+
+*****************
+ Primitive Types
+*****************
+
+Graviti SDK provides classes to initialize Portex primitive types.
```
Comment (graczhual) [inline docs/source/features/sheet_management/schema.rst:21]: use `:` instead of `.` before example

**Review by graczhual — APPROVED** [review]

**Review by AChenQ — APPROVED** [review]

**Pull request merged**